tests/python/integration/test_arm_aprofile.py

```diff
@@ -49,6 +49,7 @@ def test_conv2d(dtype):
         invar,
         weight,
         kernel_size=kernel_size,
+        channels=2,
```

Comment:
Without this change I would get an error during the AlterOpLayout pass (which is applied since we use `opt_level=3` to build the IRModule slightly below that line):
`InternalError: Check failed: (param->kernel_size.defined() && param->channels.defined()) is false: The kernel size and channels of a Conv must be set or inferred by previous pass
` which stems from this line: https://github.com/apache/tvm/blob/5e32bf0adf4b8f242bf32a3d3da598cf3083b1ac/src/relay/op/nn/convolution.cc#L1411-L1412

Reply:
I see thanks. It looks like `Conv2DGemmRel` doesn't deduce the channels while `Conv2dRel` does, hence the reason for the error. Since `Conv2DGemmRel` is used elsewhere and the channels issue hasn't been brought up before, I don't think this needs fixing here